spec/presenters/actions_presenter_spec.rb

```diff
@@ -65,19 +65,13 @@
     end
 
     context "when the document is redrafted" do
-      let(:payload) { FactoryGirl.create(:cma_case, publication_state: "redrafted") }
+      let(:payload) { FactoryGirl.create(:cma_case, first_published_at: "2016-01-01", publication_state: "draft") }
       specify { expect(subject.publish_text).to include("major edit") }
       specify { expect(subject.publish_text).to include("will email subscribers") }
     end
 
-    context "when the update_type is minor" do
-      let(:payload) { FactoryGirl.create(:cma_case, publication_state: "redrafted", update_type: "minor") }
-      specify { expect(subject.publish_text).to include("minor edit") }
-      specify { expect(subject.publish_text).not_to include("will email subscribers") }
-    end
-
     context "when the document is redrafted and the update_type is minor" do
-      let(:payload) { FactoryGirl.create(:cma_case, publication_state: "redrafted", update_type: "minor") }
+      let(:payload) { FactoryGirl.create(:cma_case, first_published_at: "2016-01-01", publication_state: "redrafted", update_type: "minor") }
```

Comment:
should this not be publication_state draft? I guess it doesn't need to change just yet, its just a little confusing in this weird still-sort-of-supporting-redrafted state